### PR TITLE
functionality to create probMaps for peakfiles from different modules [ana4; out3]

### PR DESCRIPTION
**Pull request by awirb**
* change module to str as input for probAnalysis 
* make filtering for sims optional if modName=com1DFA 
* add argument to call of runProbAnalysisOnly to allow for other comMod peakfiles to be analysed

- [x]  TODO adjust docu

### CODE REVIEW

File: avaframe/ana4Stats/probAna.py [hunk 1]
```diff
@@ -329,10 +329,10 @@ def probAnalysis(avaDir, cfg, module, parametersDict='', inputDir='', probConf='
             path to avalanche directory
         cfg : dict
             configuration read from ini file of probAna function
-        module
-            computational module that was used to run the simulations
+        modName
+            name of computational module that was used to run the simulations
```
Comment (fso42) [inline avaframe/ana4Stats/probAna.py:333]: Maybe add a note that this is the spot to address a different module?

File: avaframe/ana4Stats/probAna.py [hunk 2]
```diff
@@ -343,20 +343,24 @@ def probAnalysis(avaDir, cfg, module, parametersDict='', inputDir='', probConf='
...
-    modName = pathlib.Path(module.__file__).stem
     avaDir = pathlib.Path(avaDir)
 
     # set output directory
     outDir = avaDir / 'Outputs' / 'ana4Stats'
     fU.makeADir(outDir)
 
     # fetch all result files and filter simulations according to parametersDict
-    simNameList = cfgHandling.filterSims(avaDir, parametersDict, specDir=inputDir, simDF=simDFActual)
+    if modName == 'com1DFA':
```
Comment (fso42) [inline avaframe/ana4Stats/probAna.py:353]: Should this be lower-checked? Or do we require to always be written correctly?

File: avaframe/runProbAnalysisOnly.py [hunk 3]
```diff
@@ -17,14 +19,16 @@
...
+def runProbAna(avalancheDir="", modName="com1DFA"):
     """Run a com1DFA probability analysis with parameters and only an
     avalanche directory as input
 
     Parameters
     ----------
     avalancheDir: str
         path to avalanche directory (setup e.g. with init scripts)
+    modName: str
+        name of computational module that has been used to produce sim results
```
Comment (fso42) [inline avaframe/runProbAnalysisOnly.py:31]: Same here, quick comment about why this is here?

**Review by fso42 — CHANGES_REQUESTED** [review]

File: avaframe/runProbAnalysisOnly.py (hunk 3 above)
Comment (awirb) [reply avaframe/runProbAnalysisOnly.py:31]: added info

**Review by fso42 — APPROVED** [review]

**Pull request merged**